R/FilterState.R

```diff
@@ -279,16 +281,15 @@ FilterState <- R6::R6Class( # nolint
         tags$div(
           class = "filter-card-header",
           tags$div(
+            # header properties
             class = "filter-card-title",
             `data-toggle` = "collapse",
             `data-bs-toggle` = "collapse",
             href = paste0("#", ns("body")),
+            # header elements
+            if (private$fixed) icon("lock") else NULL,
             tags$span(tags$strong(private$get_varname())),
-            if (length(private$get_varlabel())) {
-              tags$span(private$get_varlabel(), class = "filter-card-varlabel")
-            } else {
-              NULL
-            }
+            tags$span(private$get_varlabel(), class = "filter-card-varlabel")
```

Comment:
I think it's ok. When varlabel is empty this line produces `span(character(0), class ="filter-card-varlabel)`